src/netcdf_io/calc_analysis.fd/init_calc_analysis.f90

```diff
@@ -14,17 +15,18 @@ subroutine read_nml
     !! read in namelist parameters from
     !! calc_analysis.nml file in same directory
     !! as executable
-    use vars_calc_analysis, only: anal_file, fcst_file, incr_file, use_nemsio_anl, fhr, mype, npes, jedi
+    use vars_calc_analysis, only: anal_file, fcst_file, incr_file, aero_file, use_nemsio_anl, do_aero, fhr, mype, npes, jedi
     implicit none
     ! local variables to this subroutine
     character(len=500) :: datapath = './'
     character(len=500) :: analysis_filename = 'atmanl.nc'
     character(len=500) :: firstguess_filename = 'atmges.nc'
     character(len=500) :: increment_filename = 'atminc.nc'
+    character(len=500) :: aero_inc_filename = 'aeroinc.nc'
     character(len=2) :: hrstr
     integer, parameter :: lunit = 10
     logical :: lexist = .false.
-    namelist /setup/ datapath, analysis_filename, firstguess_filename, increment_filename, fhr, use_nemsio_anl, jedi
+    namelist /setup/ datapath, analysis_filename, firstguess_filename, increment_filename, aero_inc_filename, fhr, use_nemsio_anl, do_aero, jedi
```

Comment:
Default values are set for namelist variables `fhr`, `use_nemsio_anl`, and `jedi`.  Since not all runs of `calc_analysis.x` include aerosols, we should add code to default `do_aero` to `.false.`

Reply:
Good catch, @RussTreadon-NOAA and updated!